application/features/authentication.feature

```diff
@@ -0,0 +1,135 @@
+Feature: Authentication
+  In order to identify a specific user
+  As an authorized user
```

Comment:
Thanks, that's better:  https://github.com/silinternational/idp-id-broker/commit/6365259ec1b19225759bcce43b1a8f09096e6a2a